_src/_sass/_layout.scss

```diff
@@ -128,53 +128,83 @@
     }
   }
 
-  // full screen view
+  // Use Bootstrap container behavior instead of Susy
+  // Apply container-fluid behavior manually to avoid @extend issues
```

Comment:
Now
![image](https://github.com/user-attachments/assets/9fdbeb66-04c2-42e8-adc4-e159caed4606)
- there's a white space bw header and body
- underlined links

before (expected behavior)
![image](https://github.com/user-attachments/assets/570d37eb-7618-458b-8c80-375bd2de9768)